### ISSUE TITLE
Docs : Kivy Launcher 

### ISSUE THREAD

**Issue by RobertFlatt** [kivy/kivy#7724]
The install docs reference Kivy Launcher, and installing it from the Google Play Store

https://kivy.org/doc/stable/guide/packaging-android.html#packaging-your-application-for-the-kivy-launcher

The app has apparently been removed from the Google Play Store.  The docs must reflect this.

**Comment by p0lygun** [comment]
Or it should point to 
https://play.google.com/store/apps/details?id=org.kivy.showcase

**Comment by oOperaho** [comment]
Yes, and there's no way to download it, unless you find other versions on google. 
I want to start a application without using Buildozer or generating a .exe on linux, and that's the best way i knew.

**Comment by gottadiveintopython** [comment]
You can download the latest version of apk from https://github.com/kivy/kivy-launcher (Actions tab).

**Comment by misl6** [comment]
Via #7773 now we refer to Kivy Launcher, without adding the link, so we can release 2.1.0 without a broken link.

I will keep this open until we publish it on the store, which will probably happen after 2.1.0

**Comment by Julian-O** [comment]
I don't believe publishing it in the store is a short term goal any more.

Can this be wrested from the hands of @misl6, and make it available for anyone to do?

**Comment by RobertFlatt** [comment]
This is not particularly a build problem. 
KL has a design problem, the design makes assumptions about Android shared storage that are not correct on Android >= 10.

**Comment by Julian-O** [comment]
@RobertFlatt: No comment from me, I don't understand it to that level.

My position is more caveman level: Old KL not work. New KL not work. No-one fixing. Docs not right. Kill docs.
